### PR TITLE
Replacing "Publication instance" with "user specific instance of…"

### LINKED ISSUE #1932: Should we have to define "EPUB Publication Instance"?
(This is an [editors' note in the spec](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) converted into a github issue.)

The [definition for URL's of the Root directory](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) says:

> The origin of the container root URL is unique for each EPUB Publication instance.

Do we have to define what "EPUB Publication instance" is? Or should we consider as some sort of common knowledge?

### PR DESCRIPTION
**Pull request by iherman**
Fixes #1932, see details there...

See:

* For EPUB 3.3 Reading Systems:
    * [Preview](https://raw.githack.com/w3c/epub-specs/editorial/issue-1932/epub33/rs/index.html)
    * [Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/rs/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/w3c/epub-specs/editorial/issue-1932/epub33/rs/index.html)

### CODE REVIEW

**Review by wareid — APPROVED** [review]

File: epub33/rs/index.html [hunk 1]
```diff
@@ -1172,13 +1172,13 @@ <h4>URL of the Root Directory</h4>
 							with the <a>container root URL</a> as <a data-cite="url#concept-base-url"
 								><var>base</var></a> is the <a>container root URL</a>.</li>
 						<li>The <a href="https://url.spec.whatwg.org/#origin">origin</a> of the <a>container root
-								URL</a> is unique for each EPUB Publication instance.</li>
+								URL</a> is unique for each user-specific instance of an EPUB Publication.</li>
```
Comment (rdeltour) [inline epub33/rs/index.html]: In #1932 @mattgarrish proposed:
> each user-specific instance of an EPUB Publication **in a Reading System**

The addition of "in a Reading System" implies that the the origin will not be the same for two different RS owned by the same user.

I'm not sure if it was intentionally left out or not. I don't have a definitive opinion on whether it's better with or without.

**Review by rdeltour — APPROVED** [review]
I made one non-blocking comment, but otherwise looks OK!

File: epub33/rs/index.html (hunk 1 above)
Comment (iherman) [reply epub33/rs/index.html]: Nop, it was my mistake, simply...

**Review by mattgarrish — APPROVED** [review]

**Review by lordt4ever — APPROVED** [review]

**Review by dauwhe — APPROVED** [review]
Yeah, I think this is an improvement. Thanks!

**Review by bduga — APPROVED** [review]
I don't think this is any worse and probably better than what we used to have, so lgtm. But I still find this vague concept of a user-specific instance to be fairly vague for a conformance requirement (and I am not sure how useful this section is in all honesty).

**Pull request merged**